modules/claims_api/lib/claims_api/v2/params_validation/intent_to_file/intent_to_file_validator.rb

```diff
@@ -11,6 +11,10 @@ def validate(record)
             validate_participant_vet_id(record)
           end
 
+          def self.itf_types
+            %w[compensation pension survivor].freeze
+          end
+
```

Comment:
I'd remove this and use the constant in `ClaimsApi::V2::IntentToFile`. 
As it is now, we're going to have to update both places if there is ever a change (doubtful, but future proofing).